### PR TITLE
litebrowser, litehtml: init

### PR DESCRIPTION
**Pull request by fgaz**
###### Description of changes

A simple web browser

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [x] (Package updates) Added a release notes entry if the change is major or breaking
  - [x] (Module updates) Added a release notes entry if the change is significant
  - [x] (Module addition) Added a release notes entry if adding a new NixOS module
  - [x] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/libraries/litehtml/default.nix [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  buildInputs = [
+    gumbo
+  ];
+
+  cmakeFlags = [
+    "-DEXTERNAL_GUMBO=ON"
+  ];
+
+  meta = with lib; {
+    dscription = "Fast and lightweight HTML/CSS rendering engine";
```
Comment (marsam) [inline pkgs/development/libraries/litehtml/default.nix:32]:
```suggestion
    description = "Fast and lightweight HTML/CSS rendering engine";
```

**Review by marsam — CHANGES_REQUESTED** [review]

File: pkgs/development/libraries/litehtml/default.nix (hunk 1 above)
Comment (fgaz) [reply pkgs/development/libraries/litehtml/default.nix:32]: I already fixed that in the last force push, I wonder why ofborg didn't pick it up

@GrahamcOfBorg eval

**Comment by fgaz** [comment]
I guess it has to be in a top level comment:

@GrahamcOfBorg eval

File: pkgs/development/libraries/litehtml/default.nix (hunk 1 above)
Comment (fgaz) [reply pkgs/development/libraries/litehtml/default.nix]: oh, i see. i fixed it in one package but i also copypasted the typo in the other one. fixed.

**Pull request merged**